src/main/java/org/usfirst/frc/team2473/robot/commands/AlignToHatch.java

```diff
@@ -34,18 +31,27 @@ protected void execute() {
     
     public void move() {
         double angle = Devices.getInstance().getCVAngle();
+        double distance = Devices.getInstance().getCVDistance();
+
+        double adjustAddedPower = addedPower;
+        double adjustNormalPower = normalPower;
+        if (distance < 10 && Math.abs(angle) > 10) {
```

Comment:
I don't think it's necessary to check whether `Math.abs(angle) > 10`, because if the robot is close to the target it should slow down and make more drastic turns, regardless of angle.

```suggestion
        if (distance < 10) {
```